app/models/fcrepo_endpoint.rb

```diff
@@ -11,6 +11,8 @@ def self.reset!
 
   def ping
     ActiveFedora::Fedora.instance.connection.head('/').response.success?
+  rescue Ldp::NotFound
+    ActiveFedora.fedora.connection.http.head.success?
```

Comment:
is there a reason not to just always use the fedora.connection.http.head.success? why make the clearly not complete one first?

Reply:
I'm not entirely sure it's testing the right thing. I need to have another dig at this.